src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/teiTemplates.epics.js

```diff
@@ -55,7 +55,7 @@ export const addTEITemplateEpic = (action$: InputObservable, store: ReduxStore,
                     ...(assignedUserMode && { assignedUserMode }),
                     ...(assignedUsers?.length > 0 && { assignedUsers }),
                     ...(programStatus && { enrollmentStatus: programStatus }),
-                    ...(!!followUp && { followUp }),
+                    ...(followUp !== undefined && { followUp: JSON.stringify(followUp) }),
```

Comment:
I think the same solution should be applied when [updating](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/teiTemplates.epics.js#L179) a trackedEntity WL and when [creating](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/programStageTemplates.epics.js#L66) and [updating](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/programStageTemplates.epics.js#L215) a programStage WL. WDYT?

Reply:
I'm very glad you know this logic better than me! Good catch 😄